sdcm/utils/common.py

```diff
@@ -1184,6 +1184,64 @@ def can_connect_to(ip: str, port: int, timeout: int = 1) -> bool:
     return result == 0
 
 
+def find_scylla_repo(scylla_version, dist_type='centos', dist_version=None):
+    """
+    Get a repo/list of scylla, based on scylla version match
+
+    :param scylla_version: branch version to look for, ex. 'branch-2019.1:latest', 'branch-3.1:l'
+    :param dist_type: one of ['centos', 'ubuntu', 'debian']
+    :param dist_version: family name of the distro version
+    :raises: ValueError if not found
+
+    :return: str url repo/list
+    """
+    if ':' in scylla_version:
+        branch_repo = get_branched_repo(scylla_version)
+        if branch_repo:
+            return branch_repo
+
+    repo_map = get_s3_scylla_repos_mapping(dist_type, dist_version)
+
+    for key in repo_map.keys():
+        if scylla_version.startswith(key):
+            return repo_map[key]
+    else:
+        raise ValueError("repo for scylla version {} wasn't found".format(scylla_version))
```

Comment:
Good chance to convert it to f-string :)

Reply:
the price of just copy pasting :)

Reply:
fixed